crates/storage-query-datafusion/src/inbox/schema.rs

```diff
@@ -31,5 +31,6 @@ define_table!(sys_inbox(
     sequence_number: DataType::UInt64,
 
     /// Timestamp indicating the start of this invocation.
+    /// DEPRECATED: you should not use this field anymore, but join with the sys_invocation table
```

Comment:
> Is it safer to remove it? I'll be null/0 at the moment and this might cause silent issues to anyone depending on it. We don't promise compatibility of the data fusion interface so we can remove it.

If we remove it, the CLI 1.1 will explode on commands `restate service status`, which is not so nice IMO.

Reply:
You are right. Let's keep it then.